scalyr_agent/sharded_copying_manager/copying_manager.py

```diff
@@ -799,69 +787,66 @@ def generate_status(self, warn_on_rate_limit=False):
 
             result.total_scan_iterations = self.total_scan_iterations
 
-            last_response_statuses = set()
-            health_checks = set()
+            all_health_checks_good = None
```

Comment:
Why not set this and the next one to `True`?

Reply:
Because I thought that the idea - is to not show the health check until it is not determined.